spec/human_player_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'human_player'
+require 'user_input'
+require 'fake_io'
+require 'board'
+require 'presenter'
+
+describe HumanPlayer do
+  let(:human) do
+    HumanPlayer.new(token: 'X',
+                    user_input: UserInput.new(io: FakeIO.new(input_stream: '1')))
+  end
+
+  describe '#token' do
+    it 'returns a string' do
+      expect(human.token).to be_kind_of String
+    end
+
+    it 'returns a string with one character' do
+      expect(human.token.size).to be 1
```

Comment:
Same as above, I don't think we need this test. Also what happens if I instantiate the `Human` class like this: `HumanPlayer(token: 'OO'...)` Will this test still pass?